models/staging/ecommerce/stg_order_items.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+
+models:
+  - name: stg_order_items
+    description: One unique order item per row
+    columns: 
+      - name: order_item_sk
+        description: Primary key for order item
+        tests:
+          - unique
+          - not_null
+          
```

Comment:
traditionally, we like one extra blank line at the end of files. If there is more or none you'll see the little red circle with the red line in it